region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -289,7 +290,9 @@
       "exitCondition": {
         "leaveWithSpark": {}
       },
-      "note": "Use the really small runway in the pit in the middle of the room. The door must be already opened."
+      "unlocksDoors": [{"types": ["ammo"], "requires": [], "useImplicitRequires": false}],
+      "note": "Use the really small runway in the pit in the middle of the room. The door must be already opened.",
+      "devNote": "Unlocking the door is free since obstacle A being cleared implies it was already unlocked earlier if needed."
```

Comment:
maybe we can add a [4, 4] strat that opens the door as well. Not sure what all would be required, but it's something to think about after the rework to this room is merged

Reply:
Makes sense

Reply:
looks like tricky jump + wave can open a blue door, or PBs can open a yellow one (probably tricky for obscurity)

It's also possible to open the door with a super/normal shot when getting to 4, but thats pretty tricky too